components/camel-chatscript/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
We don't use log4j.properties on component level, but using it for tests but the format must be the same with other ones